content/applications/marketing/events/attendees_reports.rst

```diff
@@ -0,0 +1,262 @@
+sub-list for that row is revealed below. To re-tuck the list, click the subsequent
+:icon:`fa-caret-down` :guilabel:`(down arrow)` beside the desired row title.
+
+Like with other reporting pages, the usual :doc:`search, filter, and group by
+<../../essentials/search>` functionality can be utilized for more customized organization, if
+needed.
+
+.. example::
+   After removing all the default filters and groupings from the :guilabel:`Attendees` reporting
+   page, while in list view, the user could click the :icon:`fa-caret-down` :guilabel:`(down arrow)`
+   to the right of the :guilabel:`Search...` bar to reveal the mega menu of :guilabel:`Filters` and
+   :guilabel:`Group By` options.
+
+   From this mega menu, the user could choose to select the :guilabel:`Registration Date`
+   :icon:`fa-caret-down` drop-down menu, and select a desired period of time to analyze: a month, a
+   quarter, or a year. In this case, they choose the current month, :guilabel:`May`. Doing so also
+   selects the current year, as well.
+
+   Then, the user could opt to choose the :guilabel:`Partner`, :guilabel:`Event`, :guilabel:`Ticket
+   Type`, and :guilabel:`Status` from the :guilabel:`Group By` column.
+
+   With that filter and groupings in place, Odoo presents the nested list of attendee data in such a
+   way that organizes the analytics from the current month, first by :guilabel:`Partner`, then by
+   :guilabel:`Event`, followed by :guilabel:`Ticket Type`, and lastly, by :guilabel:`Status`.
+
+.. _attendees_reports/cohort:
+
+Cohort view
+-----------
+
+Click the :icon:`oi-view-cohort` :guilabel:`cohort` icon in the upper-right corner to view the
+:guilabel:`Attendees` reporting metrics in a cohort table.
+
+By default, the cohort view **only** has the :guilabel:`Taken` and :guilabel:`Last 30 days` filter
+options present in the :guilabel:`Search...` bar, by default.
+
+The data displayed in the cohort table is organized by :guilabel:`Day`, by default.
+
+It should also be noted that the only option available in the :guilabel:`Measures` drop-down menu is
+:guilabel:`Count`, which is selected, by default.
+
+.. image:: attendees_reports/default-cohort-view.png
+   :align: center
+   :alt: The default cohort view of the Attendees reporting page in the Odoo Events app.
+
+On the far-left, the column shows the :guilabel:`Event Start Date`, and next to the that is the
+default :guilabel:`Count` measure option.
+
+To the right of those columns is the body of the cohort table, showcasing the :guilabel:`Create on -
+(selected time period)` data, which is :guilabel:`Day`, by default. The numbered columns below that
+show the days (via negative integers) leading up to the start of the event (represented by the
+:guilabel:`0` to the far-right).
+
+Beneath each of those days leading up to the start of the event, there is a percentage indicating
+how many tickets were :guilabel:`Taken` on that particular day (per the default settings present in
```

Comment:
```suggestion
how many tickets were :guilabel:`Taken` (or registered) on that particular day (per the default settings present in
```
I don't know if that is what the metric means- if so, it might be good to mention to the user what 'taken' is implying. I keep picturing a theater usher 'taking' my ticket and ripping it as I walk in- but these are registrants before the "0" day, so I assume it's registered. Just an idea!